Enterprise-CMCS/eAPD
# updated key state personnel

Resolves #4554 

### Description
Updated the Key State Personnel read-only to match Figma updates. Added row for key personnel total medicaid costs for MMIS.


### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| a HITECH key person | on the export view | show FFY X Cost row |  jest |
| a MMIS key person | on the export view | show FFY Cost row and Total Computable Medicaid Cost row | jest |
| a key person | on the export view | show the person's name, position, and email | jest |
| a medicaid director | on the export view | show the director's name, email, and phone number | jest |
| an empty medicaid director | on the export view | show "No Medicaid director was provided" | jest |
| a medicaid office | on the export view | show the address | jest |
| an empty medicaid office | on the export view | show "No Medicaid office was provided" | jest |
| an empty medicaid director and medicaid office | on the export view | show "No Medicaid director and corresponding Medicaid office address was provided | jest |

### Steps to manually verify this change

1. open a HITECH APD
2. go to Export and Submit page
3. click Continue to Review
4. scroll to Key State Personnel section

You won't be able to view the MMIS APD, but you can verify it on [storybook](https://www.chromatic.com/builds?appId=61d5b948cf6f17003a12bf77&branch=tforkner/4554-update-key-state-personnel)

### This pull request is ready to code review when

      tests, refactor

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
